templates/index.html

```diff
@@ -45,302 +31,234 @@
+            <p class="mt-6">
+            </p>
+          </dd>
+        </div>
+        <div class="flex flex-col">
+          <dt class="flex items-center gap-x-3 text-base/7 font-semibold text-gray-900">
+           <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="size-8 flex-none text-gray-dark">
+              <path stroke-linecap="round" stroke-linejoin="round" d="M4.5 12a7.5 7.5 0 0 0 15 0m-15 0a7.5 7.5 0 1 1 15 0m-15 0H3m16.5 0H21m-1.5 0H12m-8.457 3.077 1.41-.513m14.095-5.13 1.41-.513M5.106 17.785l1.15-.964m11.49-9.642 1.149-.964M7.501 19.795l.75-1.3m7.5-12.99.75-1.3m-6.063 16.658.26-1.477m2.605-14.772.26-1.477m0 17.726-.26-1.477M10.698 4.614l-.26-1.477M16.5 19.794l-.75-1.299M7.5 4.205 12 12m6.894 5.785-1.149-.964M6.256 7.178l-1.15-.964m15.352 8.864-1.41-.513M4.954 9.435l-1.41-.514M12.002 12l-3.75 6.495" />
+          </svg>
+           Data Strategy & Architecture
+          </dt>
+          <dd class="mt-4 flex flex-auto flex-col text-base/7 text-gray-900">
+            <p class="flex-auto">Implementation of scalable modern data infrastructure tailored to your organization's needs and budget</p>
+            <p class="mt-6">
+            </p>
+          </dd>
+        </div>
+        <div class="flex flex-col">
+          <dt class="flex items-center gap-x-3 text-base/7 font-semibold text-gray-900">
+            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor"  class="size-8 flex-none text-gray-dark">
+              <path strokeLinecap="round" strokeLinejoin="round" d="M7.5 14.25v2.25m3-4.5v4.5m3-6.75v6.75m3-9v9M6 20.25h12A2.25 2.25 0 0 0 20.25 18V6A2.25 2.25 0 0 0 18 3.75H6A2.25 2.25 0 0 0 3.75 6v12A2.25 2.25 0 0 0 6 20.25Z" />
+            </svg>
+            Analytics & Reporting
+          </dt>
+          <dd class="mt-4 flex flex-auto flex-col text-base/7 text-gray-900">
+            <p class="flex-auto">Source-of-truth metrics and actionable insights to drive strategic decision-making across your organization</p>
+            <p class="mt-6">
+            </p>
+          </dd>
+        </div>
+      </dl>
+    </div>
+  </div>
+</div>
+
+<!--FREE CONSULTATION SECTION-->
+<div class="bg-blue">
+  <div class="px-6 py-24 sm:px-6 sm:py-32 lg:px-8">
+    <div class="mx-auto max-w-4xl text-center">
+      <h2 class="text-balance text-4xl font-semibold tracking-tight text-white sm:text-5xl">Free Consultation</h2>
+      <p class="mx-auto mt-6 max-w-4xl text-pretty text-lg/8 text-white">Take our 5-minute data maturity questionnaire and we'll follow up with a custom-built consultation to inform your analytics roadmap. This puts you first in line for a free 1:1 consultation with our Data experts to discover ways to improve trust in your metrics, improve analyst efficiency and reduce infrastructure costs.</p>
```

Comment:
```suggestion
      <p class="mx-auto mt-6 max-w-4xl text-pretty text-lg/8 text-white">Take our proprietary 5-minute data maturity assessment and you'll immediately receive a highly personalized evaluation to accelerate your analytics roadmap. This puts you first in line for a free 1:1 consultation with our Data experts to discover ways to take intelligent action on your data, reduce infrastructure costs, and improve trust in your analytics.</p>
```
partially inspired back to the hero line I suggested for the landing page

Reply:
^I like most of these suggestions except:
1. It won't be instant since we're manually producing write-ups, so let's remove "immediately"
2. Kind of a nit, but I don't like the "intelligent action" wording. "Operationalize your data" could work there?